horizon_autotests/tests/test_networks.py

```diff
@@ -48,6 +48,17 @@ def test_create_shared_network(self, horizon, networks_steps):
             name=network_name).wait_for_presence()
         networks_steps.admin_delete_network(network_name)
 
+    def test_create_delete_subnet(self, horizon, networks_steps, network):
```

Comment:
@gdyuldin, `horizon` fixture usage is redundant here, because it is used already in `network` fixture.